src/tests/unit/tests/DetailsView/reports/components/report-sections/details-section.test.tsx

```diff
@@ -2,13 +2,16 @@
 // Licensed under the MIT License.
 import { shallow } from 'enzyme';
 import * as React from 'react';
-
+import { DateProvider } from '../../../../../../../common/date-provider';
 import { DetailsSection, DetailsSectionProps } from '../../../../../../../DetailsView/reports/components/report-sections/details-section';
 
 describe('DetailsSection', () => {
     it('renders', () => {
+        const scanDate = new Date(Date.UTC(2018, 2, 9, 9, 48));
```

Comment:
I extracted it out thinking I might use this for my mock; but I ended up not using it. But the extraction of this variable doesn't hurt. So, let me know if you want me to put it back to old way.